Initialize-AzureInternalEnvironments.ps1

```diff
@@ -0,0 +1,29 @@
+#######################
```

Comment:
@rnikhilkumar Remove this file please.  It is not appropriate for us to use internal endpoints in apublic repo